internal/driver/driver.go

```diff
@@ -428,21 +374,23 @@ func (d *Driver) handleWriteCommands(devName string, p protocolMap, reqs []dsMod
 		llrpResp = &llrp.CustomMessage{}
 
 	case ResourceReaderConfig:
-		data, err := params[0].StringValue()
+		/// Object value types come in as a map[string]interface{} which need to be
+		// marshaled back to JSON
+		reqData, err = json.Marshal(params[0].Value)
 		if err != nil {
 			return err
 		}
 
-		reqData = []byte(data)
 		llrpReq = &llrp.SetReaderConfig{}
 		llrpResp = &llrp.SetReaderConfigResponse{}
 	case ResourceROSpec:
-		data, err := params[0].StringValue()
+		/// Object value types come in as a map[string]interface{} which need to be
```

Comment:
change `///` to `//` here and above as well.

Reply:
removed the extra /